src/fu-tool.c

```diff
@@ -177,14 +178,17 @@ fu_util_start_engine (FuUtilPrivate *priv, FuEngineLoadFlags flags, GError **err
 	if (!fu_engine_load (priv->engine, flags, error))
 		return FALSE;
 	if (fu_engine_get_tainted (priv->engine)) {
-		g_autofree gchar *fmt = NULL;
+		/* TRANSLATORS: the user is SOL for support... */
+		msg =  _("The daemon has loaded 3rd party code and "
+			"is no longer supported by the upstream developers!");
+		fu_util_show_daemon_warning (msg);
```

Comment:
I think this is a bit of a hack; I think it either needs to be a varargs `...` style function that can accept printf-style arguments or we just use `fu_util_term_format()` and a `GString` _accumulator_.